geoengine/datasets.py

```diff
@@ -443,9 +445,16 @@ def upload_dataframe(
 
     df_json = df.to_json()
 
-    with geoengine_openapi_client.ApiClient(session.configuration) as api_client:
+    with tempfile.TemporaryDirectory() as temp_dir, \
+            geoengine_openapi_client.ApiClient(session.configuration) as api_client:
+        json_file_name = Path(temp_dir) / 'geo.json'
+        with open(json_file_name, 'w', encoding='utf8') as json_file:
+            json_file.write(df_json)
+
+        print(json_file_name, str(json_file_name))
```

Comment:
remove the print

Reply:
[4f73044](https://github.com/geo-engine/geoengine-python/pull/160/commits/4f73044a6fc77761cdb668a64de2bf63a9a6582a)